### PR TITLE
RSDK-8350 - Make ML training CLI commands clearer and update dataset file

### PR DESCRIPTION
**Pull request by tahiyasalam**

### CODE REVIEW

**Comment by etai-shuchatowitz** [comment]
Happy to go ahead and approve -- just wondering if it's more worthwhile to throw an error if the script name is malformed (i.e. doesn't match %s:%s)

File: cli/dataset.go [hunk 1]
```diff
@@ -267,7 +267,7 @@ func binaryDataToJSONLines(ctx context.Context, client datapb.DataServiceClient,
 	}
 	bboxAnnotations := convertBoundingBoxes(datum.GetMetadata().GetAnnotations().GetBboxes())
 	jsonl = ImageMetadata{
-		ImagePath:                 filenameForDownload(datum.GetMetadata()),
+		ImagePath:                 filepath.Join(dst, dataDir, filenameForDownload(datum.GetMetadata())),
```
Comment (tahiyasalam) [inline cli/dataset.go:270]: Snuck in this other change which will allow the workflows to run without any weird modifications

**Comment by tahiyasalam** [comment]
@etai-shuchatowitz RFAL! but no rush

**Review by sbal13 — APPROVED** [review]
Copy is clear, LGTM!

**Pull request merged**